src/test/resources/test_data.sql

```diff
@@ -1,4 +1,7 @@
-INSERT INTO attendance (name, campus_name)
-VALUES ('포케', '잠실');
-INSERT INTO attendance (name, campus_name, last_attended_date)
-VALUES ('무송', '잠실', CURRENT_DATE());
\ No newline at end of file
+INSERT INTO crew (name, campus) VALUES ('포케', 'JAMSIL'),
+                                       ('아루', 'JAMSIL'),
+                                    ('무송', 'SEOLLEUNG'),
+                                    ('동훈', 'SEOLLEUNG'),
+                                    ('혁수', 'SEOLLEUNG');
+INSERT INTO attendance (crew_id, date) VALUES (1, CURRENT_DATE()),
+                                              (2, CURRENT_DATE());
```

Comment:
혹시 reformat 단축키로 아래와 같이 정렬되지 않나요?
```suggestion
INSERT INTO crew (name, campus)
VALUES ('포케', 'JAMSIL'),
       ('아루', 'JAMSIL'),
       ('무송', 'SEOLLEUNG'),
       ('동훈', 'SEOLLEUNG'),
       ('혁수', 'SEOLLEUNG');

INSERT INTO attendance (crew_id, date)
VALUES (1, CURRENT_DATE()),
       (2, CURRENT_DATE());
```

Reply:
h2 문법으로 설정되지 않아서 적용되지 않았나봅니다 😅